media-libs/libvpx/libvpx-1.8.2.recipe

```diff
@@ -0,0 +1,131 @@
+SUMMARY="WebM VP8 video codec library"
+DESCRIPTION="The WebM VP8 SDK allows you to integrate your applications with \
+the VP8 video codec, a high quality, royalty free, open source codec deployed \
+on millions of computers and devices worldwide.
+
+This distribution of the WebM VP8 Codec SDK includes the following support:
+- WebM VP8 Encoder
+- WebM VP8 Decoder"
+HOMEPAGE="https://www.webmproject.org/"
+COPYRIGHT="2009-2012 Google Inc.,
+	The Mozilla Foundation and
+	The Xiph.Org Foundation"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/webmproject/libvpx/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="8735d9fcd1a781ae6917f28f239a8aa358ce4864ba113ea18af4bb2dc8b474ac"
+SOURCE_FILENAME="libvpx-$portVersion.tar.gz"
+SOURCE_DIR="libvpx-$portVersion"
+#PATCHES="libvpx-$portVersion.patchset"
```

Comment:
It builds without the patchset, if it's really not needed this line could be removed

Reply:
fixed